client/src/main/resources/ref_wsdl/WinRmService.wsdl

```diff
@@ -0,0 +1,376 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Doesn't seem like this gets updated along with the java sources? Wouldn't it get out of sync easily, leading to confusion? Perhaps better to point `java2ws` here so it gets updated, wdyt?